api/bolt/migrator/migrate_dbversion24.go

```diff
@@ -0,0 +1,12 @@
+package migrator
+
+func (m *Migrator) updateSettingsToDBVersion25() error {
+	legacySettings, err := m.settingsService.Settings()
+	if err != nil {
+		return err
+	}
+
+	legacySettings.AllowContainerCapabilitiesForRegularUsers = true
+
```

Comment:
Sorry to mess with the comments. I think we need the below code to make sure the data structure is consistent for db version 25 in both 1.24.2 and 2.0. Otherwise the below code will be missing when user updates from 1.24.2 to 2.0. We are not going to bring over the whole feature but just data structure and default values. FYI @deviantony 

```
if legacySettings.TemplatesURL == "" {
	legacySettings.TemplatesURL = portainer.DefaultTemplatesURL
}

legacySettings.UserSessionTimeout = portainer.DefaultUserSessionTimeout
```

Reply:
Hmm but the `UserSessionTimeout` do not actually exists in 1.24.1 ?